compat/react-19.mjs

```diff
@@ -1,12 +1,15 @@
-import 'preact/compat';
+import all from 'preact/compat';
 
 import { options } from 'preact';
 
 const oldVNode = options.vnode;
 options.vnode = vnode => {
-	if (typeof vnode.type === 'function') {
-		vnode.type._forwarded = true;
+	if (typeof vnode.type === 'function' && vnode.ref) {
```

Comment:
This change probably needs to be mirrored to the CJS copy (in the off chance you didn't already have that change local & just not yet pushed)